web-ui/styles/settings-panel.css

```
@@ -464,3 +464,192 @@
+    padding: 10px;
+    max-height: 260px;
+    overflow: auto;
+    border-radius: 12px;
+    border: 1px solid rgba(137, 111, 94, 0.14);
+    background: rgba(43, 37, 33, 0.94);
+    color: #f7efe7;
+    font-family: var(--font-family-mono);
+    font-size: 11px;
+    line-height: 1.5;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)